### ISSUE TITLE
[Bug]: Scrolling becomes blocked in Table element

### ISSUE THREAD

**Issue by Westbrook** [adobe/spectrum-web-components#2477]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Table

### Expected behavior

No matter the delivery context the scrollbar for an element containing an `<sp-table>` should be responsive.

### Actual behavior

In certain situations, the scrollbar for an element containing an `<sp-table>` can become unresponsive.

**Comment by hunterloftis** [comment]
@Westbrook how can I reproduce this? I have tried doing some pretty weird things to figure out which situations would break scrolling, but I can't find any:

https://studio.webcomponents.dev/edit/6M5ezA4wdUTIGYjx1t3b/src/index.ts?p=stories

- added spacers to top & bottom inside of scrolling container
- made scrolling container resizable and dynamically changed size after scrolling some amount

In what situations does the scrollbar become unresponsive?

**Comment by Westbrook** [comment]
That demo is _generally_ what goes on when it's reproduced in app, but it's still unclear how this situation is encountered.

**Comment by harprees** [comment]
Did some analysis and found that in the shared the [code](https://studio.webcomponents.dev/edit/pBhahY5fru0LyftBEhAv/src/index.ts?p=stories) `<sp-table>` is wrapped within a `<div>` with the class “container” which has the attribute overflow=”auto”, Adding <div> with the above mentioned attribute in my code base I am not seeing the scrolling issue. Did 5-10 retries.

My questions:
Is this <div> with the overflow attribute mandatory ?
Though the scroll issue seems to be resolved, I could see that the virtualizer events are not getting cleaned up. Can help explain why this might be happening and the resolution to it ?

**Comment by Westbrook** [comment]
Thanks for the added insight @harprees.

1. The specific placement of the `<div style="overflow: auto">` was just an attempt to abstract some of the page layout from your application, and while needed _somewhere_ to trigger scrolling, is not required as a direct parent. Here's an updated demo where the Table is further down the DOM tree, if that helps: https://studio.webcomponents.dev/edit/XZEYXGb1cWTttk2S889L/src/index.ts?p=stories
2. We've been tracking the event cleanup internally as it didn't seem to be causing the scrolling issues you've encountered. It looks like we've found a path towards preventing that, and assuming that's true, we plan on having an updated pre-release, soon, with the update. That the clean up doesn't happen in the above demo, either, makes me pretty sure it's a red herring, however.
3. The `.row-wrapper` element in the demo is meant to facilitate drag and drop interaction, as we've showed you in other demos.

If you could give any further information as to an abstracted reproduction case, that would help greatly. At current, after the updated pre-release we may be blocked on investigating this further for lack of information with which to work.

**Comment by harprees** [comment]
@Westbrook ,

On using the latest version of table ` - @spectrum-web-components/table@0.0.2-table.2715+208f25407`, I get error complaining about the Package path `./sp-icon is not exported `. Attaching the package.json for icon.

```
ERROR in ./src/components/asset-category-select/assets/icons.ts 19:0-47
Module not found: Error: Package path ./sp-icon is not exported from package /Users/harpreetsingh/harpreet/horizon/hz/apps/project-x/web/node_modules/@spectrum-web-components/icon (see exports field in /Users/harpreetsingh/harpreet/horizon/hz/apps/project-x/web/node_modules/@spectrum-web-components/icon/package.json)
 @ ./src/components/asset-category-select/AssetCategorySelect.ts 24:0-39 74:19-24
 @ ./src/components/asset-category-select/x-asset-category-select.ts 18:0-73 21:36-45
 @ ./src/components/editor/x-editor.ts 40:0-58
 @ ./src/editorMain.ts 101:4-90
 @ ./src/routes.ts 47:11-86
 @ ./src/initializers/routes.ts 21:0-38 29:22-31
 @ ./src/index.ts 22:0-65 90:62-86

ERROR in ./src/components/home-jumbo-sub-nav/x-home-jumbo-sub-nav.ts 21:0-47
Module not found: Error: Package path ./sp-icon is not exported from package /Users/harpreetsingh/harpreet/horizon/hz/apps/project-x/web/node_modules/@spectrum-web-components/icon (see exports field in /Users/harpreetsingh/harpreet/horizon/hz/apps/project-x/web/node_modules/@spectrum-web-components/icon/package.json)
 @ ./src/components/home-main-nav/HomeMainNav.ts 28:0-52
 @ ./src/components/home-main-nav/x-home-main-nav.ts 18:0-57 21:36-45
 @ ./src/components/app/App.ts 31:0-42
 @ ./src/components/app/x-app.ts 18:0-41 21:36-45
 @ ./src/initializers/routes.ts 31:10-87
 @ ./src/index.ts 22:0-65 90:62-86

ERROR in ./src/constants/HomeNavConstants.ts 30:0-47
Module not found: Error: Package path ./sp-icon is not exported from package /Users/harpreetsingh/harpreet/horizon/hz/apps/project-x/web/node_modules/@spectrum-web-components/icon (see exports field in /Users/harpreetsingh/harpreet/horizon/hz/apps/project-x/web/node_modules/@spectrum-web-components/icon/package.json)
 @ ./src/components/home-main-nav/HomeMainNav.ts 25:0-68 35:75-91
 @ ./src/components/home-main-nav/x-home-main-nav.ts 18:0-57 21:36-45
 @ ./src/components/app/App.ts 31:
[package.json.zip](https://github.com/adobe/spectrum-web-components/files/9289444/package.json.zip)
0-42
 @ ./src/components/app/x-app.ts 18:0-41 21:36-45
 @ ./src/initializers/routes.ts 31:10-87
 @ ./src/index.ts 22:0-65 90:62-86
```

**Comment by harprees** [comment]
With this new build `@spectrum-web-components/table@0.0.2-table.2715+208f25407` I do not see the scroll issue. Gave it a couple of retries to reproduce it.

**Comment by Westbrook** [comment]
I'm quite surprised that this corrected your issue, but that's awesome! I'll close this for now. Feel free to comment here or open a new issue if you encounter future scrolling issues.
